daemon/snapshotter/mount.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/moby/sys/mountinfo"
 )
 
-// List of known filesystems that can't be re-mounted or have shared layers
-var refCountedFileSystems = []string{"fuse-overlayfs", "overlayfs", "stargz", "zfs"}
+// List of known filesystems that can be re-mounted
+var notRefCountedFileSystems = []string{}
```

Comment:
Why not completely remove since it's not used?

Reply:
We could!

I assume that if you're asking, you don't recall any filesystem that specifically required the non-refcounted mounter? 😄

Reply:
Exactly

Reply:
Done.